python/pyspark/ml/functions.py

```diff
@@ -119,6 +122,9 @@ def array_to_vector(col: Column) -> Column:
 
```

Comment:
You should probably decorate this via `@try_remote_functions` like `functions.py`

Reply:
got it